### DISCUSSION TITLE
[JavaScript][Thunderbird] General JavaScript talk and custom userChrome.js scripts

### DISCUSSION THREAD

**Discussion by Aris-t2** [Aris-t2/CustomJSforFx#61] [General]
### Lets discuss everything related to JavaScript here instead of opening new threads for questions or request.

**Custom scripts
https://github.com/Aris-t2/CustomJSforFx/tree/master/scripts_thunderbird**

**Comment by Aris-t2** [comment]
Currently these scripts work in Thunderbird (_only tested with **method 2**_):
- CSS loader (identical in Fx and Tb)
- Custom Scrollbars (identical in Fx and Tb)
- Restart item in menubars 'File' menu and inside main application buttons popup (modified for Thunderbird)

**Comment by Janaue** [comment]
hi

I use firefox 83 and method 3 to use the script about_button.uc.js to get a button for about:config, but now the button launches the about window of firefox-mozilla.

**Comment by Aris-t2** [comment]
Looks like you are in the wrong thread. This one is about Thunderbird.

The script opens browser about window using left click and about:config tab using middle-click.
You can switch the corresponding lines within `about_button.uc.js` file, if you like.

**Comment by krystian3w** [comment]
Google started use new domain for translations:

https://github-com.translate.goog/ardiman/userChrome.js/wiki/Skriptcache?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=pl

older:

https://ooh3dpsdytm34sfhws63yjfbwy--github-com.translate.goog/ardiman/userChrome.js/wiki/Skriptcache

older (support redirect to latest):

https://translate.google.com/website?depth=1&pto=aue,boq&rurl=translate.google.com&sl=auto&sp=nmt4&tl=en&u=https://github.com/ardiman/userChrome.js/wiki/Skriptcache&usg=ALkJrhihURHiG4jaRUb5vw4aAs3Kbnkfeg

oldest (error 404):

https://translate.googleusercontent.com/translate_c?act=url&depth=1&ie=UTF8&prev=_t&rurl=translate.google.com&sl=auto&sp=nmt4&tl=en&u=https://github.com/ardiman/userChrome.js/wiki/Skriptcacheions/61
